admin/src/main/java/edu/unc/lib/dl/admin/controller/EditLabelController.java

```diff
@@ -31,7 +30,14 @@ Object saveLabel(@PathVariable("pid") String pid,
 		
 		if (label != null && label.trim().length() > 0) {
 			try {
-				this.client.modifyObject(new PID(pid), label, null, null, null);
+				PremisEventLogger logger = new PremisEventLogger(pid);
+				PID pidObject = new PID(pid);
+
+				this.client.modifyObject(pidObject, label, null, null, null);
+				
+				Element event = logger.logEvent(PremisEventLogger.Type.NORMALIZATION, "Object renamed to " + label, pidObject);
```

Comment:
We used MIGRATION as the event type for move operations, so even though that doesn't exactly match the definition of that type of event, we should switch this for consistency.